superchain/configs/configs.json

```diff
@@ -850,20 +850,27 @@
               "batcherAddr": "0x5050F69a9786F081509234F1a7F4684b5E5b76C9",
               "overhead": "0x00000000000000000000000000000000000000000000000000000000000000bc",
               "scalar": "0x00000000000000000000000000000000000000000000000000000000000a6fe0",
-              "gasLimit": 30000000
+              "gasLimit": 180000000
```

Comment:
What is this `configs.json` used for, and is it autogenerated? The old gas limit here of `30000000` has been wrong for a while, so wondering if this hints that checks are missing?

Reply:
This configs.json file is autogenerated, based on the individual chain config files. So changes to the individual chain configs get propagated to here. I still have the same question about if we want to change the `genesis.systemConfig.gasLimit` field. Or would we rather keep that static since its tied to genesis, and have a separate field to track the _current_ gasLimit